clients/privacy-center/components/modals/consent-request-modal/ConsentRequestForm.tsx

```diff
@@ -54,7 +55,7 @@ const useConsentRequestForm = ({
         // Marshall empty strings back to `undefined` so the backend will not try to validate
         email: values.email === "" ? undefined : values.email,
         phone_number: values.phone === "" ? undefined : values.phone,
-        fides_user_device_id: getFidesUserDeviceUuid(),
+        fides_user_device_id: cookie?.identity?.fides_user_device_id,
```

Comment:
is there a reason to use the `?`s? seems like they would always be defined if we are getting or creating
```suggestion
        fides_user_device_id: cookie.identity.fides_user_device_id,
```

Reply:
Yup, and in fact Typescript agrees with you too... I was just being overly nervous about it